data/json/monstergroups.json

```diff
@@ -4417,6 +4417,21 @@
       { "monster": "mon_zombie_tough", "freq": 25, "cost_multiplier": 0 }
     ]
   },
+  {
+    "type": "monstergroup",
+    "name": "GROUP_INDUSTRY",
```

Comment:
This new monster group doesn't seem to be used anywhere.

Reply:
All of these not-used-anywhere groups are used by a development branch I have, but as you mentioned those changes are not included in this PR. If you think it would make more sense to remove the groups, I . am happy to do so.

Reply:
That's a tough question... In general, especially with item groups and monster groups which are already massively bloated, it's probably best to add them in the same PR you use them.

Reply:
Removed and slated for another PR